mycroft/client/wifisetup/main.py

```diff
@@ -218,12 +402,20 @@ def connect(self, event=None):
                 connected = self.get_connected(ssid)
                 if connected:
                     wpa(self.iface, 'save_config')
-                    # ConfigurationManager.set(self.NAME, 'setup', False, True)
 
             self.client.emit(Message("mycroft.wifi.connected",
                                      {'connected': connected}))
             LOG.info("Connection status for %s = %s" % (ssid, connected))
 
+            if connected:
```

Comment:
This block is weird. This feedback should either be moved to the top connected check or the inside one.